app/src/main/java/com/g11x/checklistapp/ImportantInformationItemActivity.java

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright © 2016 Google Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ */
+
+package com.g11x.checklistapp;
+
+import android.content.Intent;
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.TextView;
+
+public class ImportantInformationItemActivity extends AppCompatActivity {
```

Comment:
Since this is open source, I think it would be helpful to potential users of the checklist framework if we put brief descriptions on the high level components of the system.